app/models/BannerDesign.scala

```diff
@@ -18,35 +18,67 @@ object BannerDesignStatus {
   implicit val statusDecoder = deriveEnumerationDecoder[BannerDesignStatus]
 }
 
+sealed trait GuardianRoundelDesign
+object GuardianRoundelDesign {
+  case object default extends GuardianRoundelDesign
+  case object brand extends GuardianRoundelDesign
+  case object inverse extends GuardianRoundelDesign
+
+  implicit val customConfig: Configuration = Configuration.default.withDefaults
+  implicit val encoder = deriveEnumerationEncoder[GuardianRoundelDesign]
+  implicit val decoder = deriveEnumerationDecoder[GuardianRoundelDesign]
+}
+
 case class BannerDesignImage(
-    mobileUrl: String,
-    tabletDesktopUrl: String,
-    wideUrl: String,
-    altText: String
```

Comment:
Oh are you using scalafmt? If so I must not have it configured to run on save for this project, sorry about that.

Reply:
I'm not - I was just making indentation consistent (2 spaces). Perhaps we should enable it for this project though